docs/configuration.md

```diff
@@ -12,10 +12,16 @@ Create `.env` file in a root directory or set environment variables with the fol
 - `STATIC_URL` - URL where the static files are located.
   E.g., if you use an S3 bucket, you should set it to the bucket's URL. By default, Saleor assumes you serve static files from the root of your site at "http://localhost:9000/".
 
-- `MARKETPLACE_URL`  - URL where Marketplace App is located. If not found, it will not render a navigation link to the Marketplace.
+- `MARKETPLACE_URL` - URL where Marketplace App is located. If not found, it will not render a navigation link to the Marketplace.
 
 - `SALEOR_APPS_PAGE_PATH` - Path appended to `MARKETPLACE_URL` to render Saleor Apps page.
 
 - `SALEOR_APPS_JSON_PATH` - Path appended to `MARKETPLACE_URL` to fetch a list of Saleor Apps as JSON.
 
 - `APP_TEMPLATE_GALLERY_PATH` - Path appended to `MARKETPLACE_URL` to render App Template Gallery page.
+
+- `FLAGS_ENABLED` - Boolean flag determines whether we use Flagsmith.
```

Comment:
please rename it to FLAGS_SERVICE_ENABLED it's more meaningful, because it's about to use external service for feature flags, not to on/off featureflags at all